setup-nextcloud/main.js

```diff
@@ -5,6 +5,7 @@ async function main() {
     try {
         const cron          = core.getInput("cron")
         const version       = core.getInput("version", {required: true})
+        const tag           = core.getInput("tag")
```

Comment:
Only one of these needs to be required. So I'd say you use version, otherwise you use tag, and if that doesn't work you error out. That stops you from duplicating the command too

Reply:
You mean remove the tag var and just use version?
That would mean though that just putting 20 or 19 won't work anymore.

The current code takes the version and appends it to "stable"

Also the pre-release makes limited sense to me in that case, user could just put master in the version list.

I would prefer that approach.

Reply:
I'm fine with that approach. I think we're the only users anyway